socketshark/__init__.py

```diff
@@ -243,13 +251,20 @@ def run(context, config):
     # Configure root logger if logging level is specified in config
     if log_config['level']:
         level = getattr(logging, log_config['level'])
-        logger = logging.getLogger()
-        logger.setLevel(level)
         formatter = logging.Formatter(log_config['format'])
         sh = logging.StreamHandler()
         sh.setFormatter(formatter)
+
+        logger = logging.getLogger(log_config['logger_name'])
```

Comment:
You are no longer configuring the root logger so this disables logging for all other Python packages